### ISSUE TITLE
Creating cluster using python docker image never returns or assigns master

### ISSUE THREAD

**Issue by willferreira** [Azure/aztk#264]
Running:

`aztk spark cluster create --id oldsparky --vm-size standard_a2 --size 2  --wait --docker-repo aztk/base:spark2.2.0-python3.6.2-base`

hangs with the following output:

```
-------------------------------------------
spark cluster id:        oldsparky
spark cluster size:      2
>        dedicated:      2
>     low priority:      0
spark cluster vm size:   standard_a2
custom scripts:          None
file shares:             0
docker repo name:        aztk/base:spark2.2.0-python3.6.2-base
wait for cluster:        True
username:                spark
-------------------------------------------

```
Moreover when I run:

`aztk spark cluster get --id oldsparky`

I can see that no master has been assigned:

```
Cluster         oldsparky
------------------------------------------
State:          active
Node Size:      standard_a2
Nodes:          2
| Dedicated:    2
| Low priority: 0

Nodes                               | State           | IP:Port              | Master  
------------------------------------|-----------------|----------------------|--------
tvm-2164905317_1-20171210t094410z   | idle            | 52.169.139.80:50001  |         
tvm-2164905317_2-20171210t094410z   | idle            | 52.169.139.80:50000  |        

```
If instead I run:

`aztk spark cluster create --id oldsparky --vm-size standard_a2 --size 2  --wait`

then the command returns and a master IS assigned. Is it just that creating a cluster using the python image is slow? I waited for 10-15 mins.

**Comment by jafreck** [comment]
The docker repository you specified for the Python image seems to be misconfigured. Try using: `aztk/python:spark2.2.0-python3.6.2-base`

All python images live under the `aztk/python` repository so I believe when the node is being set up, it is failing to find the docker image in the `aztk/base` repository.

**Comment by willferreira** [comment]
Thanks, I should have checked the repo directly. I'm pretty sure the version I was trying worked on Friday evening. Is there no error message anywhere that would tell me about the misconfigured docker image? I poked around the portal when this was going on but couldn't see any error messages relating to the image. The version you suggested works find.

Btw, the documentation [here](https://github.com/Azure/aztk/blob/master/docs/12-docker-image.md) is incorrect.

**Comment by jafreck** [comment]
Thanks for pointing out incorrect doc, we will get that fixed right away. 

And you are absolutely right -- there should be a check for the existence of the docker-image before the cluster is spun up. I raised an issue so you can track that feature (#266).

**Comment by jafreck** [comment]
Closing this since it was fixed in #265
